src/exportDataAction/ExportDataAction.js

```diff
@@ -78,9 +81,9 @@ class ExportDataAction {
     return `${sessionFilter.host}_${filterString}`;
   }
 
-  runExportTask(domainObjects) {
-    let filename = domainObjects[0].name;
-    const sessionFilter = this.sessionService.getHistoricalSessionFilter();
+  runExportTask(name, domainObjects) {
```

Comment:
Suggest the following so that you're not required to pass in a name.

```suggestion
  runExportTask(domainObjects, name) {
    let filename = name ?? domainObjects[0].name;
```